beacon-chain/p2p/service.go

```diff
@@ -649,3 +654,23 @@ func logExternalDNSAddr(id peer.ID, addr string, port uint) {
 		).Info("Node started external p2p server")
 	}
 }
+
+// Attempt to dial an address to verify its connectivity
+func verifyExternalIPAddr(addr string, port uint) {
```

Comment:
Please put this function in a more appropriate (or new) file. This file is already 600+ lines long.

Reply:
Moved to `p2p/utils.go`